README.md

```diff
@@ -9,31 +9,25 @@ OSS tools covered:
 - [gitbase](https://docs.sourced.tech/gitbase)
 - [bblfsh](https://doc.bblf.sh)
 - [BigARTM](http://bigartm.org)
-- [OpenNMT](http://opennmt.net)
+- [PyTourch](https://pytorch.org)
 
  <details>
 <summary>Abstract</summary>
 
-> Machine Learning on Source Code (MLonCode) is an emerging and exciting research domain which stands at the sweet spot between deep learning, natural language processing, social science, and programming.
+> Machine Learning on Source Code (MLonCode) is an emerging research domain which stands at the > intersection of deep learning, natural language processing, software engineering and programming > language communities.
 >
-> During this 2 hours workshop, we are going to show you how to extract insights from code bases—step by step—by shedding light on those crucial aspects:
+> During this 3.5 hours workshop, we will review the recent SE tasks that benefit from applying ML and focus the hands-on experience on:
+> - extracting data from the real source code and
+> - developing multiple different ML models
+> - for a particular task of source code summarization (or function name suggestion).
```

Comment:
```suggestion
> - summarizing source code, suggesting function names and analyzing projects and developers most relevant topics
```

Reply:
So far this text was 1:1 copy of the materials published on the website.
I'm not sure what's best to do in here

Reply:
It's probably fine to improve it here but I'm also fine keeping it as is.